### ISSUE TITLE
Internal server error with LDAP certificate

### ISSUE THREAD

**Issue by gcpmendez** [OSC/ondemand#3204]
I get the following error

![image](https://github.com/OSC/ondemand/assets/10725162/34f4c4f6-67c9-411d-855a-3f8b743815ad)

Is there any way to proceed without modifying the certificate?

**Comment by johrstrom** [comment]
> Is there any way to proceed without modifying the certificate?

I don't think so. There appears to be an environment variable, but it's unclear when that was removed and if it'd be safe to use. (all the comments about using it say it's deprecated after a certain version of go and who knows when it was removed or what version we built that binay with.)

This stack overflow topic may help:

https://stackoverflow.com/questions/64814173/how-do-i-use-sans-with-openssl-instead-of-common-name

**Comment by johrstrom** [comment]
Hey sorry to not respond again after all this time. In any case, I assume you've got this sorted out given you did not respond.

I'm going to close this now, but feel free to open more issues should you come across them or I can reopen this ticket if it persists.
